packages/front-end/components/FactTables/FactMetricModal.tsx

```diff
@@ -577,14 +591,22 @@ function getPreviewSQL({
       ? "COUNT(*)"
       : numerator.column === "$$distinctUsers"
       ? "1"
-      : `SUM(${numerator.column})`;
+      : numerator.aggregation === "count distinct"
+      ? `-- HyperLogLog estimation used instead of COUNT DISTINCT\n  COUNT(DISTINCT ${numerator.column})`
```

Comment:
I'm sure for a certain type of user this comment would be interesting, but I wonder if it will just confuse most people.  Seems like this should probably live in the docs instead where we can explain it more.

Reply:
Yeah, I waffled on this. I'll remove it.